lib/reply.js

```diff
@@ -105,7 +108,7 @@ Reply.prototype.serializer = function (fn) {
 }
 
 Reply.prototype.type = function (type) {
-  this.res.setHeader('Content-Type', type)
+  this._headers['content-type'] = type
```

Comment:
`this.header` ?

Reply:
why going for a slow-path when there is a fast-path?